src/libs/ReportActionsUtils.js

```diff
@@ -418,7 +418,11 @@ function getReportPreviewAction(chatReportID, iouReportID) {
 }
 
 function isCreatedTaskReportAction(reportAction) {
-    return reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT && _.has(reportAction.originalMessage, 'taskReportID');
+    return (
+        reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT &&
+        _.has(reportAction.originalMessage, 'taskReportID') &&
+        lodashGet(reportAction, ['originalMessage', 'html']).startsWith('Created a task: ')
```

Comment:
@thienlnam is there a better way to check differentiate created/assigned task actions besides checking the html?